### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by Kino-ki**

### CODE REVIEW

File: backend/database/schema.sql [hunk 1]
```diff
@@ -9,6 +9,16 @@ CREATE TABLE cart
   user_id    INT NOT NULL,
   product_id INT NOT NULL,
   quantity  INT NOT NULL DEFAULT 1,
```
Comment (PierreA44) [inline backend/database/schema.sql:11]: really usefull default value ?

File: backend/src/controllers/orderControllers.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+const tables = require("../tables");
+
+const browse = async (req, res, next) => {
+  try {
+    const orders = await tables.orders.readAllOrders();
+    if (orders.length) {
+      res.json(orders);
+    } else {
+      res.status(404).json({
+        message: "oops! j'ai rien à afficher",
```
Comment (PierreA44) [inline backend/src/controllers/orderControllers.js:10]: oopsy... j'ai l'impression d'entendre Pierrick !! XD

File: frontend/src/components/NavBar.jsx [hunk 3]
```diff
@@ -11,21 +11,26 @@ export default function NavBar() {
...
       title: "N liste du pere noel N",
     },
     {
       id: 4,
+      path: "/produits/commandes",
+      title: "N commandes N",
+    },
+    {
+      id: 5,
       path: "/user/inscription",
```
Comment (PierreA44) [inline frontend/src/components/NavBar.jsx:29]: franglish? user? inscription...

File: frontend/src/pages/products/OrdersPage.jsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { useLoaderData } from "react-router-dom";
+
+export default function OrdersPage() {
+  const orders = useLoaderData();
+  console.log(orders);
```
Comment (PierreA44) [inline frontend/src/pages/products/OrdersPage.jsx:5]: WTF a console.log() HERE !!!!

**Review by PierreA44 — CHANGES_REQUESTED** [review]
check my comments!

File: backend/src/controllers/orderControllers.js (hunk 2 above)
Comment (Kino-ki) [reply backend/src/controllers/orderControllers.js:10]: Pierrick Server

File: frontend/src/pages/products/OrdersPage.jsx (hunk 4 above)
Comment (Kino-ki) [reply frontend/src/pages/products/OrdersPage.jsx]: it's a sneaky console.log()

**Pull request merged**